monitoring/dr/alerts.test.yaml

```
@@ -0,0 +1,87 @@
+# These tests are meant to go hand-in-hand with the rendered alert rule.
+# Use github.com/scality/action-prom-render-test@python-renderer python module
+#
+# Render the alerts file with
+# gen-alert render alerts.yaml
+evaluation_interval: 1m
+rule_files:
+  - alerts.rendered.yaml
+tests:
+  - interval: 1m
+    input_series:
+      - series: up{drSinkInstance="", job="artesca-data-dr-source-base-queue-connector-metrics", namespace="zenko"}
+        values:  _x7
+      - series: up{drSinkInstance="artesca-data-dr", job="artesca-data-dr-base-queue-connector-metrics", namespace="zenko"}
+        values:  1x7
+      - series: kafka_connect_task_error_total_record_errors{drSinkInstance="", job="artesca-data-dr-source-base-queue-connector-metrics"}
+        values:  0 1 2 2 2 2 2
+      - series: kafka_connect_task_error_total_record_failures{drSinkInstance="", job="artesca-data-dr-source-base-queue-connector-metrics"}
+        values:  0x7
+      - series: kafka_connect_task_error_total_record_errors{drSinkInstance="artesca-data-dr", job="artesca-data-dr-base-queue-connector-metrics"}
+        values:  0 1 2 2 2 2 2
+      - series: kafka_connect_task_error_total_record_failures{drSinkInstance="artesca-data-dr", job="artesca-data-dr-base-queue-connector-metrics"}
+        values:  0 1x6
+      - series: s3_lifecycle_last_timestamp_ms{drSinkInstance="", job="artesca-data-backbeat-lifecycle-.*-headless", namespace="zenko"}
+        values:  10000000 10000000 10000000
+      - series: mongodb_ss_repl_lastWrite_lastWriteDate{drSinkInstance="artesca-data-dr", job="zenko/data-db-mongodb-sharded-shard.*", namespace="zenko"}
+        values:  5000000 5000000 5000000
+    alert_rule_test:
+      - alertname: DrResourcePausedWhileOtherRunning
+        eval_time: 1m
+        exp_alerts:
+          - exp_labels:
+              severity: warning
+            exp_annotations:
+              summary: 'DR Resource Paused While Other Running'
+              description: 'One site s DR resource is paused while the other is not. This could lead to data inconsistency between sites.'
```

Maybe we can rewrite the description of the alert like this?
```suggestion
              description: 'One site\'s DR resource is paused while the other is not. This could lead to data inconsistency between sites.'
```

Also, should this one become a critical alert after X hours/days?

---

We can do that but we should determine the duration , or better yet as we're checking if a site's dr resource is paused for too long isn't it kind of the same thing ?